specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/examples/PrivateEndpointConnections_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,84 @@
+{
+  "title": "PrivateEndpointConnections_Create",
+  "operationId": "PrivateEndpointConnections_Create",
+  "parameters": {
+    "subscriptionId": "48841AC8-BAB7-41CD-A060-F539444ACD50",
+    "resourceGroupName": "rgelasticsan",
+    "elasticSanName": "1jRHU_51x_8f_142",
+    "privateEndpointConnectionName": "djwitfgttbuofvxnju",
+    "parameters": {
+      "properties": {
+        "privateEndpoint": {},
+        "privateLinkServiceConnectionState": {
+          "status": "Pending",
```

Comment:
Should we input status in create private endpoint?
Normally status should be returned from server.

Reply:
There is provisioning status in ElasticSan, VolumeGroup and Volume so i followed this pattern. Let me know if it is concerning.

Reply:
Just would like to make sure if customer really need to input status as "Pending" in create privateEndpoint?
If so, this is OK for me.
If user should not input it, but the value should be from server, we should change the sample.